### PR TITLE
Add .teamMember module

### PR DESCRIPTION
**Pull request by Tara-Tiong**
# teamMember PR 
> Mobile media query
Screen Shot 2020-09-17 at 5 20 45 PM

> Desktop version

Screen Shot 2020-09-17 at 5 22 44 PM

### CODE REVIEW

File: storybook/index.html [hunk 1]
```diff
@@ -138,6 +138,24 @@ <h1 class="sectionHeading">Hello</h1>
...
+                </div>
+                <div class="teamMember--role">
+                    BAMP Executive Director/ <br> Lead Artist
+                </div>
+                <div class="teamMember--linkedin">
+                    <img src="../images/linkedin.svg">
+                </div>
+                <div class="teamMember--mail">
+                    <img src="../images/mail.svg">
+                </div>
```
Comment (johncip) [inline storybook/index.html]: think we're missing a closing div tag here

**Comment by johncip** [comment]
hey, almost there, let's tweak to be closer to Figma -- there's a lighter color for the role, it's not italic, and some more vertical space between things, and the name could be a little bigger.

**Comment by Tara-Tiong** [comment]
# Updated PR 
I didn't add `gridBox.css`
Screen Shot 2020-09-17 at 7 50 28 PM
Screen Shot 2020-09-17 at 7 49 27 PM

**Review by johncip — APPROVED** [review]
looks good! :shipit:

**Comment by johncip** [comment]
@Tara-Tiong I just noticed a tiny thing (and pushed a commit for it) -- `teamMember--linkedin` and `teamMember--mail` had the same code, so we can use the same class for each. I deleted one and renamed them to `teamMember--icon`. Gonna merge now.

**Comment by johncip** [comment]
@Tara-Tiong @kingtrentiii @Ernesto2004 just a reminder, if you'd like to get rid of references to branches that have been deleted on github, the command is

`git fetch --prune`

(and it's `git prune` for local branches)

**Pull request merged**